README.md

```diff
@@ -13,81 +13,78 @@ Uses SponsorBlock data licensed used under CC BY-NC-SA 4.0. More details: https:
 ![image](images/example.png)
 
 ## Setup/Installation
-1. Install Python 3.7 or greater from https://www.python.org/
-    - Mac/Linux users, you might already have python 3 installed on your computer. Open your terminal and type `python`. If your command prompt says `Python 3.7` or higher you do not need to install from the website.
-    - Mac/Linux users, your `python` command might use Python 2, which is not supported. If this is the case use the command `python3` instead of `python` for the rest of this setup. 
-        
+1. Install required software.  
+    -  Install Python 3.7 or greater from https://www.python.org/
+    
+    *For Windows users*, if you do not bash already installed, you can download Git Bash [here]((https://git-scm.com/download/win))
```

Comment:
Bash is not necessary needed in windows, only git. This download comes with git bash but primarily is for installing git.